### DISCUSSION TITLE
Q: 部署后不显示评论时间 || Q: The comment time is not displayed after deployment

### DISCUSSION THREAD

**Discussion by whcxxb** [walinejs/waline#2222] [Q&A] [answered]
### Describe the bug | 问题描述

有大佬帮忙看下吗
![WX20231227-105444@2x](https://github.com/walinejs/waline/assets/20062144/84eef264-8a66-477a-b8f0-9dd9c0cdbacf)
![WX20231227-105512@2x](https://github.com/walinejs/waline/assets/20062144/773b77bf-dfcd-4755-9e9c-c6c91e58a8df)

### Website URL | 问题网站

https://www.wxxb.cc/posts/2023-9-8

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Can anyone help me take a look?
![WX20231227-105444@2x](https://github.com/walinejs/waline/assets/20062144/84eef264-8a66-477a-b8f0-9dd9c0cdbacf)
![WX20231227-105512@2x](https://github.com/walinejs/waline/assets/20062144/773b77bf-dfcd-4755-9e9c-c6c91e58a8df)

### Website URL | Problem website

https://www.wxxb.cc/posts/2023-9-8

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
@Mister-Hope Hopes 老师，我看 `@waline/client@3` 的版本应该是启用到了新的 API 接口格式 `/api/xxx` 了，时间这块看是不是要适配下，按照之前约定的服务端下发成时间戳格式了，换了个 `time` 字段。
![image](https://github.com/walinejs/waline/assets/424491/b2a90482-4d55-497f-89ae-564ff788fe22)

**Comment by Mister-Hope** [comment, marked as answer]
已修复

---
Fixed
